src/bsoncxx/document/value.hpp

```diff
@@ -23,6 +23,11 @@
 
 namespace bsoncxx {
 BSONCXX_INLINE_NAMESPACE_BEGIN
+
+namespace array {
+class view;
+}
```

Comment:
Could this be an include?
```
#include <bsoncxx/array/view.hpp>
```

Reply:
Since you have the include now you should be able to delete this.

Reply:
Flagging this comment again.

Reply:
The forward declaration should already be removed